src/components/Microsoft.DotNet.UpgradeAssistant.MSBuild/VisualStudioFinder.cs

```diff
@@ -35,35 +35,34 @@ public void Configure(WorkspaceOptions options)
 
         private (string? Path, int? Version) GetLatestVisualStudioPath(string? suppliedPath)
         {
-            var latest = GetLatestPath(suppliedPath);
+            var latest = GetLatestVisualStudio(suppliedPath);
 
-            if (latest is null)
+            if (latest.InstallPath is null)
             {
                 _logger.LogWarning("Did not find a Visual Studio instance");
                 return default;
             }
 
-            var version = Version.Parse(latest.GetInstallationVersion());
-            var installation = latest.GetInstallationPath();
+            _logger.LogError("InstallationPath:{InstallPath}  Version:{Version}", latest.InstallPath, latest.Version);
```

Comment:
Should this be removed? There's no error here and you log this information later as well

Reply:
d'oh, yes. Thanks.